project-settings/organizations/organization-settings.md

```diff
@@ -33,75 +33,82 @@ The Security tab is only available for the **Plus plan** or above
 
 This tab gathers all security options of your Organization. For now you can only configure Single Sign-On (SSO).
 
-#### Configuring SSO
+![](<../../.gitbook/assets/organization-settings-security.png>)
+
+
+### Configuring SSO
 
 {% hint style="info" %}
-We are officially supporting: **Okta, OneLogin, Google** and **Azure** IdPs.
+The are supporting the SAML 2.0 specifications, you can use all the Identity providers you want therefore we recommend to use on of the following officially supported platforms.
```

Comment:
I am not sure of the message here: are we supporting all SAML 2.0 ? Or are we saying that we only support the 4 below ?

Reply:
Both